### ISSUE TITLE
Batch Rendering Request

### ISSUE THREAD

**Issue by boricuapab** [coolzilj/Blender-ControlNet#7]
Would be great to have the ability to batch render like in A1111 batch tab in img2img.  At the moment I'm rendering 1 frame at a time. to get a result similar to this.

![Uploading girlJump.gif…]()

**Comment by boricuapab** [comment]
https://drive.google.com/file/d/1DT5km94J65vmcJ0rUwZCiLZP7hoTZqi5/view?usp=share_link

**Comment by moon47usaco** [comment]
I agree. being able to hit "render animation" and having it run on each frame would be what I am looking for.

I am using this now but being able to render from blender would be awesome.
https://github.com/fishslot/video_loopback_for_webui

Then being able to send different passes (color, depth...) to separate nets.
